javascript/big-practice/src/scripts/models/noteModel.js

```diff
@@ -191,31 +131,20 @@ export default class NoteModel {
    * @param {String} title is title of note
    * @param {String} description is description of note
    *
-   * @returns {Object} this.notes[noteIndex]
+   * @returns {Object} noteItem
    */
   async editNote(id, title, description) {
```

Comment:
Currently, we just edit the title and description in a note. But in the future, we want to change many things in a note, so how can we do that?